build.gradle

```
@@ -173,6 +175,7 @@ configure(project(':common')) {
     }
 
     dependencies {
+        implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.41"
```

```suggestion
        implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
```

would you consider introducing a version variable for kotlin?

---

done